### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by flowed**
```
kong/kong/cache_warmup.lua:88:28: "intented" is a misspelling of "intended"
kong/kong/db/dao/plugins.lua:163:15: "strat" is a misspelling of "start"
kong/kong/db/dao/plugins.lua:165:57: "strat" is a misspelling of "start"
kong/kong/db/dao/targets.lua:160:42: "overriden" is a misspelling of "overridden"
kong/kong/db/dao/targets.lua:184:42: "overriden" is a misspelling of "overridden"
kong/kong/db/schema/init.lua:1827:52: "agianst" is a misspelling of "against"
kong/kong/db/schema/others/migrations.lua:1:6: "strat" is a misspelling of "start"
kong/kong/db/schema/others/migrations.lua:11:63: "strat" is a misspelling of "start"
kong/kong/db/schema/others/migrations.lua:12:63: "strat" is a misspelling of "start"
kong/kong/db/schema/plugin_loader.lua:86:23: "unkown" is a misspelling of "unknown"
kong/kong/db/strategies/cassandra/init.lua:1186:56: "intergration" is a misspelling of "integration"
kong/kong/db/strategies/init.lua:58:17: "strat" is a misspelling of "start"
kong/kong/db/strategies/init.lua:59:59: "strat" is a misspelling of "start"
kong/kong/pdk/router.lua:45:63: "targetted" is a misspelling of "targeted"
kong/kong/pdk/service/request.lua:35:71: "ocurred" is a misspelling of "occurred"
kong/kong/pdk/service/request.lua:36:56: "ocurred" is a misspelling of "occurred"
kong/kong/pdk/service/request.lua:272:50: "occurences" is a misspelling of "occurrences"
kong/kong/templates/kong_yml.lua:44:36: "relationshp" is a misspelling of "relationships"
kong/kong/tools/utils.lua:731:57: "occurence" is a misspelling of "occurrence"
kong/kong/tools/utils.lua:804:3: "Explictly" is a misspelling of "Explicitly"
kong/spec/01-unit/01-db/01-schema/01-schema_spec.lua:3483:22: "foriegn" is a misspelling of "foreign"
kong/spec/01-unit/01-db/01-schema/01-schema_spec.lua:3488:31: "foriegn" is a misspelling of "foreign"
kong/spec/01-unit/01-db/01-schema/01-schema_spec.lua:3535:22: "foriegn" is a misspelling of "foreign"
kong/spec/01-unit/01-db/01-schema/01-schema_spec.lua:3540:31: "foriegn" is a misspelling of "foreign"
kong/spec/02-integration/03-db/02-db_core_entities_spec.lua:922:47: "explicitely" is a misspelling of "explicitly"
kong/spec/02-integration/03-db/07-tags_spec.lua:99:43: "seperately" is a misspelling of "separately"
kong/spec/02-integration/03-db/07-tags_spec.lua:152:43: "seperately" is a misspelling of "separately"
kong/spec/02-integration/03-db/07-tags_spec.lua:189:43: "seperately" is a misspelling of "separately"
kong/spec/02-integration/04-admin_api/03-consumers_routes_spec.lua:272:18: "emtpy" is a misspelling of "empty"
kong/spec/02-integration/05-proxy/14-server_tokens_spec.lua:729:27: "occured" is a misspelling of "occurred"
kong/spec/03-plugins/01-tcp-log/01-tcp-log_spec.lua:185:35: "milisecond" is a misspelling of "millisecond"
kong/spec/03-plugins/01-tcp-log/01-tcp-log_spec.lua:222:35: "milisecond" is a misspelling of "millisecond"
kong/spec/03-plugins/01-tcp-log/01-tcp-log_spec.lua:259:35: "milisecond" is a misspelling of "millisecond"
kong/spec/03-plugins/02-udp-log/01-udp-log_spec.lua:114:35: "milisecond" is a misspelling of "millisecond"
kong/spec/03-plugins/02-udp-log/01-udp-log_spec.lua:150:35: "milisecond" is a misspelling of "millisecond"
kong/spec/03-plugins/02-udp-log/01-udp-log_spec.lua:186:35: "milisecond" is a misspelling of "millisecond"
kong/spec/03-plugins/13-cors/01-access_spec.lua:780:20: "orgin" is a misspelling of "origin"
kong/spec/03-plugins/13-cors/01-access_spec.lua:900:29: "orgin" is a misspelling of "origin"
kong/spec/03-plugins/22-aws-lambda/01-access_spec.lua:761:29: "overriden" is a misspelling of "overridden"
kong/spec/03-plugins/23-rate-limiting/04-access_spec.lua:271:13: "Additonal" is a misspelling of "Additional"
kong/spec/03-plugins/23-rate-limiting/04-access_spec.lua:300:13: "Additonal" is a misspelling of "Additional"
kong/spec/03-plugins/23-rate-limiting/04-access_spec.lua:677:11: "Additonal" is a misspelling of "Additional"
kong/spec/03-plugins/23-rate-limiting/04-access_spec.lua:735:11: "Additonal" is a misspelling of "Additional"
kong/spec/03-plugins/23-rate-limiting/04-access_spec.lua:792:11: "Additonal" is a misspelling of "Additional"
kong/spec/03-plugins/24-response-rate-limiting/04-access_spec.lua:356:9: "Additonal" is a misspelling of "Additional"
kong/spec/fixtures/ssl.lua:85:74: "Intermidiate" is a misspelling of "Intermediate"
kong/spec/helpers.lua:1015:62: "alternativly" is a misspelling of "alternatively"
```

### CODE REVIEW

File: kong/templates/kong_yml.lua [hunk 1]
```diff
@@ -41,7 +41,7 @@ _format_version: "1.1"
 # consumers:
 # - username: example-user
 #   # Custom entities from plugin can also be specified
-#   # If they specify a foreign-key relationshp, they can also be nested
+#   # If they specify a foreign-key relationships, they can also be nested
```
Comment (gszr) [inline kong/templates/kong_yml.lua]: This is also wrong - "a foreign-key relationship"

File: kong/db/strategies/init.lua [hunk 2]
```diff
@@ -55,8 +55,8 @@ function _M.new(kong_config, database, schemas, errors)
       return nil, nil, err
     end
 
-    local custom_strat = fmt("kong.db.strategies.%s.%s", database, schema.name)
-    local exists, mod = utils.load_module_if_exists(custom_strat)
+    local custom_start = fmt("kong.db.strategies.%s.%s", database, schema.name)
```
Comment (gszr) [inline kong/db/strategies/init.lua]: This is incorrect - "strat" comes from "strategy", please revert.

File: kong/db/schema/others/migrations.lua [hunk 3]
```diff
@@ -1,4 +1,4 @@
-local strat_migration = {
+local start_migration = {
```
Comment (gszr) [inline kong/db/schema/others/migrations.lua]: This is incorrect - "strat" comes from "strategy", please revert.

File: kong/db/dao/plugins.lua [hunk 4]
```diff
@@ -160,9 +160,9 @@ local function load_plugin_entity_strategy(schema, db, plugin)
     return nil, err
   end
 
-  local custom_strat = fmt("kong.plugins.%s.strategies.%s.%s",
+  local custom_start = fmt("kong.plugins.%s.strategies.%s.%s",
```
Comment (gszr) [inline kong/db/dao/plugins.lua]: This is incorrect - "strat" comes from "strategy", please revert.

**Comment by yoverflow** [comment]
@gszr can you make code suggestion? so we can commit it easily!

**Comment by hishamhm** [comment]
@flowed thank you for this! which tool you used to spell check the code? Instead of making multiple code suggestions/commits, you could set `strat` as an allowed exception in the tool and just run it again.

**Pull request closed without merging**